src/FileConverter.ts

```diff
@@ -266,11 +265,10 @@ export class FileConverter {
             throw new Error("File converter properties were not properly initialized.");
         }
 
-        if (!FileConverter.multiTool) {
-            FileConverter.multiTool = FileConverter.extensionContext.asAbsolutePath("/resources/sarif.multitool/Sarif.Multitool.exe");
-        }
-
-        return FileConverter.multiTool;
+        // I'd love to use the type-script "usage" here, but that causes a type-script error
+        // https://www.npmjs.com/package/@microsoft/sarif-multitool
+        // so, I'm just doing what the node package does internally (yuck).
+        return  FileConverter.extensionContext.asAbsolutePath(`node_modules/@Microsoft/sarif-multitool-${process.platform}/Sarif.Multitool.exe`);
```

Comment:
Per the readme, we should be using the default export (which is a string representing the path) instead of hardcoding the path. The path may not be guaranteed in the future.

I realize you had issues with `esModuleInterop` but we should try to resolve those properly.

Reply:
Yep. Done.